i18n/locales/source/hourofcode/en.yml

```diff
@@ -36,6 +36,8 @@
 
   hoc2018_tutorial_mchoc_description: 'Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.'
 
+  codeorg_homepage_hoc2019_dance_heading: 'Try the new <span class="highlight-word">Dance&nbsp;Party!</span>'
```

Comment:
@breville is it possible to do this without using raw html? We generally want to avoid consuming HTML from translated strings so we don't have to worry about translators adding unwanted HTML.

Reply:
Thanks.  As discussed, this is tracked.